lib/shared/puppet_self_service.rb

```diff
@@ -1,6 +1,6 @@
 require 'puppet'
 
-# PuppetSelfService - Shared code for Puppet Self Service facts
```

Comment:
We should rename this as well. To do that, just rename the file and replace all instances of `PuppetSelfService` with `PEStatusCheck` across every file in the repo.